package.json

```diff
@@ -78,8 +75,5 @@
     "tslint": "5.16.0",
     "typescript": "3.4.5",
```

Comment:
old typescript

Reply:
Typescript >= 4.x makes tests unable to run, and I have yet to figure out a fix. I am intending to bump this in another PR, specifically https://github.com/microsoft/powerbi-visuals-mekkochart/pull/96

Reply:
So, is there any progress with it?